pep-0622.rst

```diff
@@ -169,6 +169,9 @@ The proposed indentation structure is as following::
         case pattern_2:
             ...
 
+Here, `some_expression` represents the value that is being matched against,
+which will be referred to hereafter as the *subject* of the match.
```

Comment:
Why not change the example to say `match subject:` and leave it at that? The approximate grammar above already says it's an expression.

Reply:
Because people might think we mean the literal word 'subject'. Saying 'some_expression' makes it clear that it's a placeholder.

You could also use `match <subject>` but then the example is no longer syntactically correct.

Reply:
Okay, fair. I'll merge.